tests.py

```diff
@@ -237,16 +246,19 @@ def test_get_tld_returns_correct_tld_or_etld(self):
         assert 'co.uk' == psl.get_tld('co.uk', wildcard=True)
         assert 'co.uk' == psl.get_tld('co.uk', wildcard=False)
         assert None == psl.get_tld('blah.local', strict=True)
-        assert None == psl.get_tld('blah.local', wildcard=False)
+        # non-strict mode: TLD = 'local'
+        assert 'local' == psl.get_tld('blah.local', wildcard=False)
         assert 'local' == psl.get_tld('blah.local')
 
     def test_get_tld_returns_correct_tld_or_etld_for_fqdn(self):
         psl = publicsuffix.PublicSuffixList()
-        assert 'com' == psl.get_tld('www.foo.com.')
+        # note: empty label or dot on the right side is not allowed
```

Comment:
i thought that was just for the unsafe version of the method? get_tld() would strip but unsafe would not.